app/models/megaphone/episode.rb

```diff
@@ -178,6 +178,12 @@ def check_audio!
       end
     end
 
+    def audio_is_latest?
+      feeder_audio_updated_at = feeder_episode.media_versions.last&.updated_at
+      megaphone_updated_at = DateTime.parse(audio_file_updated_at) if audio_file_updated_at
+      megaphone_updated_at && feeder_audio_updated_at && feeder_audio_updated_at <= megaphone_updated_at
+    end
```

Comment:
so the `original_filename` is getting set to a guid, not the value send over by the API.
I believe this is how it is when uploading via the API, when uploading via the site, I believe this is the correct filename.

So, we can't rely on that - instead, use the updated_at of the media version, and expect that the audio on megaphone has to be updatd after that, or it is out of sync.

Not perfect, but better than nothing to check to see the audio updated - if this proves to be unreliable, could start tracking the timestamp of the audio even more.

Reply:
Assuming _some_ lag in megaphone processing, this seems like it would be good enough.

Reply:
I'm assuming lag in our processing, between when we cut a version, and when that is available on DTR. and when we then send it megaphone as a new audio file